BE: Create APIs about lesson

hauphanlvc opened the pull request.

hauphanlvc committed:
BE: Create APIs about lesson

Signed-off-by: hauphanlvc <hauphanlvc@gmail.com>

AkiraNoob commented on backend/middleware/permissionAccess.ts:
```diff
@@ -24,6 +24,25 @@ export const userRolePermissionMiddleware =
     return;
   };
 
+export const adminRolePermissionMiddleware =
```
tui nghĩ là ông k cần cái hàm này, ở cái route dùng hàm này sao mình k gọi userRolePermissionMiddleware([EUserRole.Admin])

hauphanlvc replied:
Đã thay thế adminRolePermissionMiddleware bằng userRolePermissionMiddleware([EUserRole.Admin]) và mọi thứ chạy lại ổn. Cảm ơn ông góp ý

AkiraNoob replied:
nếu thay rồi thì remove cái hàm này ông, vì k dùng nữa

AkiraNoob committed:
add document type

AkiraNoob reviewed (commented):
cái types document hay, tui có sửa mấy cái cũ của tui theo cái types này

hauphanlvc committed:
NOREF:
+ userRolePermissionMiddleware([EUserRole.Admin]) instead of adminRolePermissionMiddleware.
+ check flow after applying document types.

Signed-off-by: hauphanlvc <hauphanlvc@gmail.com>
